src/managed/OpenLiveWriter.PostEditor/PreferencesHandler.cs

```diff
@@ -150,13 +150,12 @@ private static void LoadPreferencesPanels()
             preferencesPanelTypeTable["accounts"] = type;
             types.Add(type);
 
-            //ToDo: OLW Spell Checker
             //if (SpellingSettings.CanSpellCheck)
```

Comment:
Could uncomment or delete this check for `if (SpellingSettings.CanSpellCheck) `, I could go either way.